platforms/documentation/docs/src/docs/userguide/dep-man/basics/dependency_resolution.adoc

```diff
@@ -0,0 +1,307 @@
+- Each version of a *module* is referred to as a *component*, such as link:https://mvnrepository.com/artifact/com.fasterxml.jackson.core/jackson-databind/2.17.2[`com.fasterxml.jackson.core:jackson-databind:2.17.2`].
+
+A *project* contributes a single component to the dependency graph, which itself belongs to a module.
+
+In the example below, the *component* `com.fasterxml.jackson.core:jackson-databind:2.17.2` is added as a dependency to the `implementation` *configuration* in a Java application:
+
+[source,kotlin]
+.build.gradle.kts
+----
+dependencies {
+    implementation("com.fasterxml.jackson.core:jackson-databind:2.17.2")
+}
+----
+
+=== 2. Perform conflict resolution
+
+image::dep-man-basics-3.png[]
+
+Gradle identifies and resolves any version conflicts when multiple declared or transitive dependencies request different *versions* of the same *module*.
+
+Even though a user might declare version `2.17.2` of a *module*, this may not be the version ultimately resolved in the graph.
+Gradle’s conflict resolution strategy, which defaults to selecting the highest version, selects a single version of a module when multiple are requested.
+
+However, Gradle APIs can be used to change the outcome:
+
+- **Resolution Rules**: Gradle allows configuring rules to enforce specific versions, reject certain versions, or substitute dependencies as needed.
+- **Dependency Substitution**: Rules defined in build logic can replace one dependency with another, alter versions, or redirect requests for one module with another.
+- **Dynamic Versions**: If dependencies are defined with dynamic versions (e.g., `1.0.+`) or version ranges (e.g., `[1.0, 2.0)`), Gradle resolves these to specific versions by querying the repositories.
+- **Dependency Locking**: If enabled, Gradle checks lock files to ensure consistent versions across build invocations, preventing unexpected changes in dependency versions.
+
+In the example, Gradle selects the *component* `com.fasterxml.jackson.core:jackson-databind:2.17.2` (the `2.17.2` version of the `com.fasterxml.jackson.core:jackson-databind` *module*).
+
+=== 3. Retrieve the metadata
+
+image::dep-man-basics-4.png[]
+
+Once Gradle has determined which version of an external *module* to resolve, it fetches the *metadata* for the *component* from an `ivy`, `pom`, or link:https://repo1.maven.org/maven2/com/fasterxml/jackson/core/jackson-databind/2.17.2/jackson-databind-2.17.2.module[`GMM`] metadata file in the repository.
+
+Here’s a sample of the *metadata* for link:https://repo1.maven.org/maven2/com/fasterxml/jackson/core/jackson-databind/2.17.2/jackson-databind-2.17.2.module[`com.fasterxml.jackson.core:jackson-databind:2.17.2`]:
+
+[source,json]
+----
+{
+  "formatVersion": "1.1",
+  "component": {
+    "group": "com.fasterxml.jackson.core",
+    "module": "jackson-databind",
+    "version": "2.17.2",
+  },
+  "variants": [
+    {
+      "name": "apiElements"
+    },
```

Comment:
```suggestion
    {
      "name": "apiElements",
      ...
    },
```

I think this would make it more clear that we're leaving out information